services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/core/application.py

```diff
@@ -164,12 +167,7 @@ async def _on_shutdown() -> None:
         if docker_compose_yaml := app_state.compose_spec:
             logger.info("Removing spawned containers")
 
-            result = await docker_compose_down(
-                docker_compose_yaml,
-                app.state.settings,
-                # NOTE: in the event of a SIGTERM, there is a limited time to cleanup
-                timeout=app.state.settings.DYNAMIC_SIDECAR_DOCKER_COMPOSE_DOWN_TIMEOUT,
-            )
+            result = await docker_compose_down(docker_compose_yaml, app.state.settings)
```

Comment:
just saw you also removed the NOTE. so what happens in case of SIGTERM or SIGKILL?

Reply:
The note made no sense. Please ignore its existance. I put it there when testing this and it remain in the code.